app/assets/stylesheets/application.css.scss

```diff
@@ -13,5 +13,18 @@
 // Customize
 @import "content";
 @import "style";
+
+// Controller
 @import "pages";
 @import "search";
+@import "settings";
+
+.flash {
+  margin: 0 10px 20px;
+  padding: 10px;
+
+  &.error {
+    border: solid 1px #f99;
```

Comment:
Color literals like `#f99` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered background-color, border